.github/workflows/entrypoint.sh

```diff
@@ -13,7 +13,7 @@ mkdir -p /var/lock/
 
 opkg update
 
-[ -n "${CI_HELPER:=''}" ] || CI_HELPER="/ci/.github/workflows/ci_helpers.sh"
+CI_HELPER="${CI_HELPER:="/ci/.github/workflows/ci_helpers.sh"}"
```

Comment:
Is this intended?

Reply:
I probably should have left a seperate commit for this, but yes.

if `CI_HELPER` was null or empty it would be set to the literal 2 char string `''`.  
so
```bash
[ -n "''" ]
```
would always be greater than zero chars long (2) and never call the command after the `||`

Reply:
Removed this line to alleviate any contention since it was a benign bug anyway